### ISSUE TITLE
i18n WG Meeting - March 13th

### ISSUE THREAD

**Issue by obensource** [nodejs/i18n#223]
### Date & Time
* 6:00pm UTC on March 6, 2020

### Links
* [Zoom link](https://zoom.us/j/307660555)
* [Google Docs Minutes](https://docs.google.com/document/d/1_X3Ybw5nRC8vKU2zfcValo-Kj0y76GpAfYEMJtGUaS0/edit?usp=sharing)

### Agenda

Please feel free to add or subtract issues as is relevant. 👍

#### l10n
* l10n team structure mirroring Crowdin roles: Owners (l10n team champions), Reviewers, Translators (not an issue yet, but worth discussing).
* Localization best practices guide, #214
* Update our Guides, #210
* Create Crowdin Glossary, #81, #91

#### i18n
* Unblock translation process, [#2906](https://github.com/nodejs/nodejs.org/issues/2906)
  * Do we need to create another [special Node.js Github User](https://github.com/nodejs/i18n/issues/57), configure the @nodejs-crowdin bot to handle multiple tokens via crowdin (is that possible), or migrate to using the @nodejs/github-bot ?
* Publish a module to npm, #92
* Website content to be translated, #72
* Automatic approve and merge translation PRs, #161

#### Intl
* Onboarding new Intl contributors (not yet an issue)
  * What kind of help is needed the most in Intl right now?

**Note: Please feel free to add any issues you'd like to discuss today.** 

### Invited Members
* @nodejs/i18n
* @nodejs/website-redesign
* @nodejs/community-committee
* @nodejs/tsc
* @nodejs/website

**Comment by obensource** [comment]
Unfortunately, I'm currently experiencing a family health emergency and won't be able to facilitate or attend today's meeting.

If anyone is willing to facilitate – I can likely hop on to start the youtube stream. If there's no attendance, no problem. We can reconvene next week. :)

Thanks all! 🙏

**Comment by maddhruv** [comment]
@obensource take care in this prime time, I hope everyone is healthy

Also, I also wont be able to attend

**Comment by srl295** [comment]
I'm going to have to drop out this time. But I am getting some things moved forward on the locale data building front. Also ICU 66.1 shipped

**Comment by nodejs-crowdin** [comment]
I'm @zeke masquerading as @nodejs-crowdin 👋 

I generated a new token with the following scopes: https://github.com/settings/tokens/new?scopes=repo,user,admin:repo_hook&description=Crowdin%20integration

...and added it to the https://crowdin.com/project/nodejs GitHub integration.

cc @alexandrtovmach

**Comment by JohnTitor** [comment]
It seems @nodejs-crowdin triggers many CI builds?

**Comment by zeke** [comment]
Hi @JohnTitor what are you referring to? Can you share more details?

**Comment by JohnTitor** [comment]
@zeke See here: https://travis-ci.com/github/nodejs/i18n/builds
Currently, travis receives every branch commits.

**Comment by zeke** [comment]
Yeah maybe there's a way to use Travis cron jobs for this, instead of running on every commit. Or we could use GitHub Actions. Can you open a separate issue to look into this?
